artifacts/definitions.py

```diff
@@ -22,24 +25,32 @@
     'Containerd': 'Containerd artifacts',
     'Docker': 'Docker artifacts.',
     'Execution': 'Contain execution events.',
-    'ExternalAccount': (
+    'ExternalAccount': '<Deprecated>', # TODO: to be deleted
+    'External Account': (
         'Information about any user accounts e.g. username, '
         'account ID, etc.'),
     'External Media': 'Contain external media data or events e.g. USB drives.',
+    'File System': 'Artifacts related to the file system e.g. MFT.',
     'Hadoop': 'Hadoop artifacts.',
     'IM': 'Instant Messaging / Chat applications artifacts.',
+    'Interactive': 'Artifacts created by interactive (e.g. RDP) user activity.',
     'iOS': 'Artifacts related to iOS devices connected to the system.',
     'History Files': 'History files artifacts e.g. .bash_history.',
-    'KnowledgeBase': 'Artifacts used in knowledge base generation.',
+    'KnowledgeBase': '<Deprecated>', # TODO: to be deleted
+    'Knowledge Base': 'Artifacts used in knowledge base generation.',
     'Kubernetes': 'Kubernetes artifacts',
     'Logs': 'Contain log files.',
     'Mail': 'Mail client applications artifacts.',
     'Memory': 'Artifacts retrieved from memory.',
     'Network': 'Describe networking state.',
+    'Persistence' : 'Persistence mechanisms e.g. the Startup folder.',
     'Plist': 'Artifact that is a plist.',
     'Processes': 'Describe running processes.',
     'Rekall': 'Artifacts using the Rekall memory forensics framework.',
-    'Software': 'Installed software.',
+    'Registry': 'Files related to the Windows Registry.',
```

Comment:
What is the purpose of this label?

Reply:
I admit that maybe there's not much value in having a label for the registry files (hives, transaction files, backups, etc.). It just felt right to also assign them a label :) but I'm also happy to delete this label. WDYT?

Reply:
If we're not using it remove it